app/resources/linked_account_resource.rb

```diff
@@ -0,0 +1,17 @@
+class LinkedAccountResource < BaseResource
+  model_hint model: LinkedAccount::MyAnimeList
```

Comment:
I wanna say I think the class name needs to be in a string, like `"LinkedAccount::MyAnimeList"`

I'm not sure, check ListImportResource